doc/source/gotchas.rst

```diff
@@ -1,18 +1,92 @@
 .. currentmodule:: pandas
 .. _gotchas:
 
+********************************
+Frequently Asked Questions (FAQ)
```

Comment:
Ah, sorry that is a leftover from the other PR. I changed everything from FAQ back to gotchas, apart from the header.